src/main/scala/esl/FSConnection.scala

```diff
@@ -902,6 +902,72 @@ abstract class FSConnection extends StrictLogging {
             })
             .mkString("\n")}""".stripMargin
         )
+      case (Some(appId), Some(commandToQueue), _, Some(EventNames.ChannelExecuteComplete)) =>
```

Comment:
case (
            Some(appId),
            Some(
              CommandToQueue(command: Record, executeEvent, executeComplete)
            ),
            _,
            Some(EventNames.ChannelExecuteComplete)
          ) if appId == command.eventUuid =>

Reply:
use this format

Reply:
@vincent-seville